speechmatics/cli_parser.py

```diff
@@ -467,6 +467,10 @@ def get_arg_parser():
         help="Which type of diarization to use.",
     )
 
+    rt_transcribe_command_parser.add_argument(
+        "--audio-events", action="store_true", help="Print audio events to the console"
```

Comment:
What constitutes an audio event?

Reply:
This is a good point, you need a bit more exposition in the Python client help, so people don't have to resort to the main documentation.